torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1326,7 +1369,14 @@ RegisterOperators reg2({
           "[] l, int start, int end=9223372036854775807, int step=1) -> " decl_type \
           "[]",                                                                     \
           listSlice<Shared<c_type>, c_type::ElemType>),                             \
-      Operator("aten::list(" decl_type "[] l) -> " decl_type "[]", listList)
+      Operator("aten::list(" decl_type "[] l) -> " decl_type "[]", listList),       \
```

Comment:
I think all of these should have the same alias set for the output list as the input list

Reply:
listMulIntLeft & listMulIntRight both look good now! "aten::list(" decl_type "[] l)" needs aliasing as well i think

Reply:
Turns out you were fine the first time, sorry! Since we treat any values taken in or of a data structure as a wildcard, for the current implementation we don't need to worry when the contents of one list get copied to the contents of another list. We're only tracking mutation to the data structures themselves...